ddp-workspace/projects/ddp-pancan/src/app/components/faq-section/faq-section.component.html

```diff
@@ -0,0 +1,22 @@
+<mat-accordion hideToggle="true" multi="true" displayMode="flat">
+    <ng-container *ngFor="let item of questions | translate; let i=index;">
+        <ng-container
+            *ngTemplateOutlet="question; context: { item: item, show: (questionsCount && i < questionsCount) || !questionsCount }">
+        </ng-container>
+    </ng-container>
+</mat-accordion>
+
+<ng-template #question let-item="item" let-show="show">
```

Comment:
#minor just wondering what's the benefit of moving this part of the html into seprate ng-template here?

Reply:
just a workaround: separating a conditional logic from a view one.
Fixed at 5934f4245f5e114a6153ffdb7e957e24f6eb646a